events/events.go

```diff
@@ -0,0 +1,308 @@
+package events
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/service-fabrik-cli-plugin/errors"
+	"github.com/SAP/service-fabrik-cli-plugin/helper"
+	"github.com/cloudfoundry/cli/plugin"
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	httpClientDisabledSecurityCheck *http.Client
+	httpClientEnabledSecurityCheck  *http.Client
+)
+
+type EventCommand struct {
+	cliConnection plugin.CliConnection
+}
+
+func initialize() {
+	httpClientDisabledSecurityCheck = createHttpClient(true)
+	httpClientEnabledSecurityCheck = createHttpClient(false)
+}
+
+func NewEventsCommand(cliConnection plugin.CliConnection) *EventCommand {
+	command := new(EventCommand)
+	command.cliConnection = cliConnection
+	return command
+}
+
+const (
```

Comment:
If you are creating a constants file, move constants present in individual files, like this, to the constants file.